changelog_entry.yaml

```
@@ -0,0 +1,5 @@
+
+- bump: minor
```

Since it doesn't introduce new functionality; see https://semver.org
```suggestion
- bump: patch
```